apl-api/src/main/java/com/apollocurrency/aplwallet/api/dto/FbcPropertyDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.apollocurrency.aplwallet.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class FbcPropertyDTO {
```

Comment:
Later you can change 'FbcPropertyDTO' name into more suitable. Now it can be like that for the first PR.